tests/main/prepare-image-grub/task.yaml

```diff
@@ -39,46 +41,47 @@ execute: |
     fi
 
     echo Expose the needed assertions through the fakestore
-    cp $TESTSLIB/assertions/developer1.account $STORE_DIR/asserts
-    cp $TESTSLIB/assertions/developer1.account-key $STORE_DIR/asserts
+    cp "$TESTSLIB"/assertions/developer1.account "$STORE_DIR/asserts"
+    cp "$TESTSLIB"/assertions/developer1.account-key "$STORE_DIR/asserts"
     # have snap use the fakestore for assertions (but nothing else)
     export SNAPPY_FORCE_SAS_URL=http://$STORE_ADDR
 
     echo Running prepare-image
+    #shellcheck disable=SC2086
```

Comment:
Is this one needed? The line below looks okay to me and removing it and running shellcheck (0.5.0) seems to be fine.